app/src/main/java/org/fossasia/openevent/app/data/models/Attendee.java

```diff
@@ -30,44 +33,46 @@
 
 @Data
 @Builder
+@Type("attendee")
 @AllArgsConstructor
-@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
 @EqualsAndHashCode(callSuper = false)
 @Table(database = OrgaDatabase.class)
 public class Attendee extends AbstractItem<Attendee, AttendeeViewHolder> implements Comparable<Attendee>, IHeaderProvider {
-
+    @Id(LongIdHandler.class)
     @PrimaryKey
     public long id;
-
     @Column
-    public boolean checkedIn;
+    public String city;
     @Column
-    public String email;
+    public String firstname;
     @Column
-    @JsonProperty("firstname")
-    public String firstName;
+    public String lastname;
     @Column
-    @JsonProperty("lastname")
-    public String lastName;
-    @ForeignKey(
-        saveForeignKeyModel = true,
-        deleteForeignKeyModel = true,
-        onDelete = ForeignKeyAction.CASCADE,
-        onUpdate = ForeignKeyAction.CASCADE)
-    public Order order;
+    public boolean isCheckedIn;
+    @Column
+    public String state;
+    @Column
+    public String address;
+    @Column
+    public String pdfUrl;
+    @Column
+    public String country;
+    @Column
+    public String email;
 
-    /**
-     * The ticket itself should not be deleted if the Attendee is deleted
-     * Neither it should be inserted when inserting the attendee, but the
-     * model should load instantly with attendee, making the relationship
-     * NOT stubbed.
-     */
+    @Relationship("ticket")
     @ForeignKey(onDelete = ForeignKeyAction.CASCADE)
     public Ticket ticket;
 
+    // Not in API yet
```

Comment:
No. It isn't. Event-invoice is completely different.
https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md#event-invoices---eg-v1event-invoice